cleanlab_studio/internal/clean_helpers.py

```diff
@@ -53,3 +53,44 @@ def poll_cleanset_status(
         if res["has_error"]:
             pbar.set_postfix_str(res["step_description"])
             raise CleansetError(f"Cleanset {cleanset_id} failed to complete")
+
+
+def validate_label_column(
```

Comment:
this is acceptable for now, but note that this is not a scalable way to handle validation. for every additional validation check that we add, we'll have to add an API method + a call to it from our client.

instead, we should centralize these validation methods (a `validate_project` endpoint) that contains all of the logic. then we can update that as desired without needing to make corresponding client changes every time.

Can you create a task to do so? this was the original intention we had when deciding to add matching validation to the Python client, but it wasn't specified for you here. I don't want to block this PR longer than necessary, but we should make this change later on

Reply:
Yeah that's a good point. Created a task [here](https://www.notion.so/cleanlab/Move-project-creation-validation-to-backend-ee78e332554644e0a436b2daee610290?pvs=4)